_dropin-templates/dropin-container.mdx

```diff
@@ -0,0 +1,176 @@
+---
+title: ContainerOne container
+description: Learn about the ContainerOne container and configurations for the DROPIN_NAME drop-in component.
+sidebar:
+  label: ContainerOne
+  order: 2
+---
+
+import OptionsTable from '@components/OptionsTable.astro';
+import { Steps } from '@astrojs/starlight/components';
+
+{/* DESCRIBE what the ContainerOne container does and its main purpose for the DROPIN_NAME drop-in component. Include key capabilities like customization, instances, and automatic handling of states. */}
+
+## Container rendering
+
+<Diagram caption="ContainerOne container">
+  ![ContainerOne container](@images/placeholder.webp)
+</Diagram>
+
+## Import
+
+```javascript
+import ContainerOne from '@dropins/storefront-DROPIN_PACKAGE/containers/ContainerOne.js';
+```
+
+## Configurations
+
+The `ContainerOne` container provides the following configuration options:
+
+<OptionsTable
+  compact
+  options={[
+    ['Option', 'Type', 'Req?', 'Description'],
+    ['OPTION_NAME', 'TYPE', 'Yes/No', 'DESCRIPTION of what this option does and its default value if applicable'],
+    ['OPTION_NAME', 'TYPE', 'Yes/No', 'DESCRIPTION of what this option does and its default value if applicable'],
+  ]}
+/>
+
+## Basic usage
+
+```javascript
+await provider.render(ContainerOne, {
+  OPTION_NAME: VALUE_OR_FUNCTION,
+  OPTION_NAME: VALUE_OR_FUNCTION,
+  onCallback: (data) => {
+    // DESCRIBE what the callback does with the data
+  }
+})($container);
+```
+
+## Slots
```

Comment:
I see that product-discovery has a separate `slots` page: https://experienceleague.adobe.com/developer/commerce/storefront/dropins/product-discovery/slots/

But I think it makes more sense to document as they are a facet of the container.

Do you have plans to refactor existing docs to match this template format?